nextflow.config

```
@@ -32,6 +32,7 @@ params {
 
     // General Options
     memory_usage_log_deep = false
+    maximum_process_num = 1000
```

```suggestion
    max_task_num = 1000
```
It's actually for the same process, but with different instances, i.e. tasks in Nextflow